keras/backend/mxnet_backend.py

```diff
@@ -167,9 +167,12 @@ def cast_to_floatx(x):
 
 
 def is_sparse(tensor):
-    """
-    MXNet backend do not yet support sparse tensor operations.
-    """
+    if isinstance(tensor, KerasSymbol):
+        if isinstance(_forward_pass(tensor)[0], mx.ndarray.sparse.CSRNDArray) or \
+                isinstance(_forward_pass(tensor)[0], mx.ndarray.sparse.RowSparseNDArray):
+                return True
+    elif hasattr(tensor, 'tocoo'):
```

Comment:
nit: but a very minor perf improvement if you swap these if elif . hasattr(..) faster than isinstance, forward_pass etc.

Reply:
done